lib/gemika/matrix.rb

```diff
@@ -45,6 +56,53 @@ def validate!
         contents.include?('gemika') or raise UnusableGemfile, "Gemfile is missing gemika dependency: #{gemfile}"
       end
 
+      private
+
+      ##
+      # Checks if there is the requested ruby version is aliased by rbenv to use another ruby version.
+      # Returns the runnable ruby version.
+      #
+      def aliased_ruby(requested_version, current_version)
+        ruby_aliases = rbenv_aliases
+
+        aliased_versions = {}
+
+        ruby_aliases.split("\n").each do |ruby_alias|
+          split_pattern = /\A(.+) => (.+)\z/
+          alias_name, aliased_version = ruby_alias.match(split_pattern)&.captures
+          aliased_versions[alias_name] = aliased_version
+        end
+
+        find_aliased_ruby(requested_version, current_version, aliased_versions)
+      end
+
+      ##
+      # Recursively traverses aliases until the requested Ruby version is found.
+      # Returns the requested version if no alias can be found for that version.
+      #
+      def find_aliased_ruby(requested_version, current_version, aliased_versions)
+        found_version = aliased_versions[requested_version]
+
+        if found_version == requested_version
```

Comment:
shouldn't this be compared to the `current_version`? 
The expression is only true if
`aliased_versions[requested_version] == requested_version`, which means this would be a self reference in an alias. That's probably not possible.
Also `current_version` is never used anywhere.